src/main/java/org/glovo/glovo/controller/OrderController.java

```diff
@@ -0,0 +1,51 @@
+package org.glovo.glovo.controller;
+
+import lombok.RequiredArgsConstructor;
+import org.glovo.glovo.model.Order;
+import org.glovo.glovo.model.Product;
+import org.glovo.glovo.service.OrderService;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+@RestController
+@RequestMapping("/api/orders")
+@RequiredArgsConstructor
+public class OrderController {
+    private final OrderService orderService;
+
+    @GetMapping
+    public List<Order> getAllOrders() {
+        return orderService.getAllOrders();
+    }
+
+    @GetMapping("/{id}")
+    public Order getOrderById(@PathVariable Long id) {
+        return orderService.getOrderById(id);
+    }
+
+    @PostMapping
+    public Order createOrder(@RequestBody Order order) {
```

Comment:
it's better to use separate model like CreateOrderRequest for create, update etc